test/functional/feature_minchainwork.py

```diff
@@ -82,7 +82,7 @@ def run_test(self):
         msg.hashstop = 0
         peer.send_and_ping(msg)
         time.sleep(5)
-        assert "headers" not in peer.last_message
+        assert ("headers" not in peer.last_message or len(peer.last_message["headers"].headers) == 0)
```

Comment:
Nit: braces are unnecessary, I think.

Reply:
Done as part of 8efa73e7ce4ea0c1b7ad5c3947a5ecf9fb6361d3